### PR TITLE
Measurements Widget: content needs some padding on sides

### PR DESCRIPTION
**Pull request by wilmaier**
As in the Properties widget, some padding on the sides of the content is needed.

### CODE REVIEW

**Comment by wilmaier** [comment]
image


image

File: packages/itwin/measure-tools/src/ui-2.0/MeasurementPropertyWidget.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+div[data-item-id='measure-tools-property-widget'] {
```
Comment (aruniverse) [inline packages/itwin/measure-tools/src/ui-2.0/MeasurementPropertyWidget.scss:6]: why using data-item id and not an actual class?

**Review by aruniverse — COMMENTED** [review]
plz provide rush change file

File: common/changes/@itwin/measure-tools-react/feat-measure-widget-padding_2023-05-24-15-30.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/measure-tools-react",
+      "comment": "Content in measurements widget needs some padding",
+      "type": "none"
```
Comment (aruniverse) [inline common/changes/@itwin/measure-tools-react/feat-measure-widget-padding_2023-05-24-15-30.json]:
```suggestion
      "type": "patch"
```

**Pull request merged**